### PR TITLE
Terraform 0.6.2

### PR DESCRIPTION
**Pull request by mitto**

### CODE REVIEW

**Comment by DomT4** [comment]
Test is failing here:

```
==> /usr/local/Cellar/terraform/0.6.2/bin/terraform plan -var aws.region=us-west-2 /private/tmp/terraform20150807-37617-1ehf6jd
Refreshing Terraform state prior to plan...

[31mError refreshing state: 1 error(s) occurred:

* 1 error(s) occurred:

* InvalidClientTokenId: The security token included in the request is invalid.
	status code: 403, request id: [be10658a-3d33-11e5-9906-3d7e7d075f04][0m[0m
Error: terraform: failed
```

**Comment by mitto** [comment]
@DomT4 I tried to fix failed test. review please.

File: Library/Formula/terraform.rb [hunk 1]
```diff
@@ -134,34 +134,6 @@ def install
...
-      }
-
-      resource "aws_instance" "web" {
-        instance_type = "m1.small"
-        ami = "${lookup(var.aws_amis, var.aws_region)}"
-        count = 4
-      }
-    EOS
-    system "#{bin}/terraform", "plan", "-var", "aws.region=us-west-2", testpath
+    system "#{bin}/terraform", "version"
```
Comment (dunn) [inline Library/Formula/terraform.rb]: It passes now, but we'd love a test that does a little more in terms of checking functionality.

Comment (mitto) [inline Library/Formula/terraform.rb]: @dunn Thank you for review. I tried to fix it again.

**Comment by dunn** [comment]
Looks good, thanks! Merged in 50d384458ce76dd08b0ea51b2840f412212eba5b.

**Comment by mitto** [comment]
@dunn Thank you for merge! :smile:

**Pull request closed without merging**